platforms/software/dependency-management/src/main/java/org/gradle/internal/component/local/model/DefaultLocalComponentGraphResolveState.java

```diff
@@ -95,7 +96,7 @@ public DefaultLocalComponentGraphResolveState(long instanceId, LocalComponentMet
     @Override
     public void reevaluate() {
         configurations.clear();
-        getArtifactMetadata().reevaluate();
+        getMetadata().reevaluate();
```

Comment:
❗ Is this correct?  This used to return artifact metadata now it returns `graphMetadata` from the abstract base class.

Reply:
Yeah this is correct. If you look at the prior version of this class, you can see we passed `metadata, metadata` into the super constructor of `AbstractComponentGraphResolveState`. Both `getArtifactMetadata()` and `getMetadata()` returned the same thing. 

In reality the split between graph metadata and artifact metadata does not really make much sense for local components, so that distinction only exists for external components now.